packages/web-twig/src/Resources/components/VisuallyHidden/__tests__/__fixtures__/visuallyHiddenDefault.twig

```diff
@@ -0,0 +1,4 @@
+<VisuallyHidden>Label</VisuallyHidden>
+
+<!-- Render as div element -->
+<VisuallyHidden elementType="div">Label</VisuallyHidden>
```

Comment:
New line at the end of the file is missing… Where is Prettier? 😨 

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/3dee43ba-6c3d-4500-ab71-115c4287dfba)

Reply:
You mean `insert_final_newline = true` in `.editorconfig`? There is no such setting. Also see https://github.com/prettier/prettier/issues/6360#issuecomment-518994475. Do we want this?